docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -101,20 +101,15 @@ It is not recommended to run scalable mode with `filesystem` storage. For the pu
      bloomGateway:
        replicas: 0
 
+      # This exposes the Loki gateway so it can be written to and queried externaly
+     gateway:
+        service:
+          type: LoadBalancer
+
+
      # Enable minio for storage
      minio:
        enabled: true
-
```

Comment:
I agree with removing the zeroed out replicas, as I think that's no longer needed since the introduction of `deploymentMode`. However, should we provide values for the microservice targets here, even if it's just a copy of the defaults, for clarity to the user?

Reply:
Aaah good point! I have added it back in but kept it omitted from the AWS tutorial. From testing, this is fine using `deploymentMode` as you pointed out.